README.md

```diff
@@ -47,25 +47,11 @@ to use:
 
 ## Using libtock-rs
 
+### Within this project (as example)
 The easiest way to start using libtock-rs is adding an example to the examples folder.
-The boiler plate code you would write is
-```rust
-#![no_std]
-
-extern crate tock;
-
-fn main() {
-  // Your code
-}
-```
-If you want to use heap based allocation you will have to add
-```rust
-#![feature(alloc)]
-extern crate alloc;
-```
-to the preamble.
+Just copy an example for the quickstart.
```

Comment:
If we added this in favour of explaining how to use the library I would miss the part about heap based allocation.